app/cron.js

```diff
@@ -290,6 +297,15 @@ class CronManager {
         'Communication with database (mongoDB) was established.'
       );
 
+      const api = CronManager.createAxios();
+      const tenants = await api
+        .get('http://auth:5000/admin/tenants')
+        .then((response) => response.data.tenants);
+
+      for (const tenant of tenants) {
+        this.setTenant(tenant);
```

Comment:
Hm... naming a function 'SetTenant' doesn't explain very well what this function does, maybe createTenantManagment... I don't know, but I would think about it rs.

Reply:
Done.